contributing/topics/guide-new-resource.md

```diff
@@ -637,6 +653,128 @@ func (ResourceGroupExampleResource) expandComplexResource(input []ComplexResourc
 
 - Historically, we used `pluginsdk.StateChangeConf` to address certain issues related to LRO APIs. This method has now been deprecated and replaced by custom pollers. Please refer to this [example](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/maps/custompollers/maps_account_poller.go).
 
+- Argument names, `model` and `properties` must be wrapped in backticks in error messages and validation messages.
```

Comment:
Is this covered by https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-documentation-standards.md?